runbotics/runbotics-orchestrator-ui/src/src-app/views/tenants/TenantsListView/TenantPluginsDrawer/TenantPluginsView/TenantPluginsView.styles.ts

```diff
@@ -0,0 +1,63 @@
+import { Drawer, Stack } from '@mui/material';
+import styled from 'styled-components';
+
+export const PluginDrawerTitle = styled('h4')(({theme}) => ({
+    padding: '16px 24px',
+    fontFamily: 'Roboto',
+    fontSize: theme.typography.pxToRem(20),
+    fontWeight: 500,
+}));
+
+export const PluginExpDate = styled('p')(() => ({
+    fontFamily: 'Roboto',
+    fontSize: '14px',
+    fontWeight: 400,
+}));
+
+export const PluginDate = styled('span')<{
+    expired?: boolean;
+    expiringSoon?: boolean;
+}>`
+    fontFamily: 'Roboto';
+    fontSize: '14px';
+    fontWeight: 400;
+    color: ${({ expired, expiringSoon }) =>
+        // eslint-disable-next-line no-nested-ternary
+        expired ? '#aaa' : expiringSoon ? '#863034' : 'inherit'};
+`;
+
+export const PluginBadge = styled('span')<{
+    expired?: boolean;
+    expiringSoon?: boolean;
+}>`
+    font-family: Roboto;
+    font-size: 0.75rem;
```

Comment:
here we use rem, and before we used px? Under what circumstances do we use each?